cli/module_loader.rs

```diff
@@ -63,6 +63,34 @@ impl CliModuleLoader {
     &self,
     specifier: &ModuleSpecifier,
   ) -> Result<ModuleCodeSource, AnyError> {
+    if specifier.as_str() == "node:module" {
+      eprintln!("wants node:module");
+
+      let source = r#"
+const m = Deno[Deno.internal].require.moduleExports;
+export const _cache = m._cache;
+export const _extensions = m._extensions;
+export const _findPath = m._findPath;
+export const _initPaths = m._initPaths;
+export const _load = m._load;
+export const _nodeModulePaths = m._nodeModulePaths;
+export const _pathCache = m._pathCache;
+export const _preloadModules = m._preloadModules;
+export const _resolveFilename = m._resolveFilename;
```

Comment:
I should probably make it `module_es_shim.js` and use `include_str!()` in `ext/node` instead of writing it inline here